app/services/feature_toggle.rb

```diff
@@ -25,7 +25,7 @@ def self.bulk_upload_lettings_logs?
   end
 
   def self.bulk_upload_sales_logs?
-    !Rails.env.production?
+    true
```

Comment:
This is an important change so should be in a separate PR.

Reply:
No problem